libr/util/r_asn1.c

```diff
@@ -41,16 +41,16 @@ RASN1String *r_asn1_concatenate_strings (RASN1String *s0, RASN1String *s1, bool
 	str = (char*) malloc (len);
 	if (!str) {
 		if (freestr) {
-			r_asn1_free_string (s0);
-			r_asn1_free_string (s1);
+			RASN1_STR_FREE0 (s0);
```

Comment:
no need to use macros for that, the pointers are not used after they are freed, so its innecessary